spec/cf_app_discovery/parser_spec.rb

```diff
@@ -41,7 +43,8 @@
           state: "STARTED",
           detected_start_command: ""
         },
-        route: "route-3",
+        hostname: "route-3",
+        path: "",
```

Comment:
Probably doesn't matter but we could mix up the test cases for path so it isn't always the empty string

Reply:
why? it's not this test's job to test that behaviour